egs2/voxceleb/spk1/conf/tuning/train_rawnet3_sgdr_bs.yaml

```diff
@@ -25,33 +32,35 @@ preprocessor_conf:
   noise_apply_prob: 0.0
   rir_apply_prob: 0.0
 
+# Model conf
 model_conf:
   extract_feats_in_collect_stats: false
 
+# Loss
 loss: aamsoftmax
 loss_conf:
-  nout: 256
-  nclasses: 7205
   margin: 0.3
   scale: 30
 
+# Training related
 max_epoch: 40
-#num_iters_per_epoch: 1
 num_att_plot: 0
-num_workers: 8
+num_workers: 0
```

Comment:
more close to a question. Do we normally set default num_workers to 0 in cfgs?

Reply:
It depends on the CPU cores attached, but I found that using multiple one would cause issue in collect stats stage. But still, it might not be necessary to keep 0, I will change it back.